engine/src/main/java/org/entando/entando/aps/system/services/api/server/ApiRestStatusServer.java

```diff
@@ -65,28 +97,29 @@ public Object getApiStatus(@PathParam("httpMethod") String httpMethodString,
                 response.setResult(ApiStatus.FREE.toString(), null);
             }
         } catch (ApiException ae) {
-            response.addErrors(((ApiException) ae).getErrors());
+            response.addErrors(ae.getErrors());
             response.setResult(ApiStatus.INACTIVE.toString(), null);
         } catch (Throwable t) {
             return this.buildErrorResponse(httpMethod, namespace, resourceName, t);
         }
-        return response;
+        return ResponseEntity.ok(response);
     }
 
-    private StringApiResponse buildErrorResponse(ApiMethod.HttpMethod httpMethod,
-			String namespace, String resourceName, Throwable t) {
-		StringBuilder buffer = new StringBuilder();
-		buffer.append("Method '").append(httpMethod).
-				append("' Namespace '").append(namespace).append("' Resource '").append(resourceName).append("'");
-		_logger.error("Error building api response  - {}", buffer.toString(), t);
+    private ResponseEntity<StringApiResponse> buildErrorResponse(ApiMethod.HttpMethod httpMethod,
+            String namespace, String resourceName, Throwable t) {
+        StringBuilder buffer = new StringBuilder();
```

Comment:
why not use a simple and more readable `String.format`?

Reply:
done